src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/PhysicalProjectTreeStorageTests.cs

```diff
@@ -108,27 +108,27 @@ public void CreateFolderAsync_WhenTreeNotPublished_ThrowsInvalidOperation()
             string result = null;
             var treeProvider = IProjectTreeProviderFactory.ImplementFindByPath((root, path) => { result = path; return null; });
             var currentTree = ProjectTreeParser.Parse(projectPath);
-            var service = IProjectTreeServiceFactory.Create();
-            var physicalProjectTree = IPhysicalProjectTreeFactory.Create(treeProvider, currentTree, service);
-
-            var storage = CreateInstance(physicalProjectTree: physicalProjectTree, unconfiguredProject: unconfiguredProject);
+            
+            var storage = CreateInstance(treeProvider:treeProvider, unconfiguredProject: unconfiguredProject);
 
             await storage.CreateFolderAsync(input);
 
             Assert.Equal(expected, result);
         }
 
-        private PhysicalProjectTreeStorage CreateInstance(IPhysicalProjectTree physicalProjectTree = null, IFileSystem fileSystem = null, IFolderManager folderManager = null, UnconfiguredProject unconfiguredProject = null)
+        private PhysicalProjectTreeStorage CreateInstance(IProjectTreeService treeService = null, IProjectTreeProvider treeProvider = null, IFileSystem fileSystem = null, IFolderManager folderManager = null, UnconfiguredProject unconfiguredProject = null)
```

Comment:
nit: Move each parameter to new line. Easier to read with line starting at single tab from the previous line indentation